swagger.json

```diff
@@ -224,21 +240,79 @@
     },
     "File" : {
       "type" : "object",
-      "required" : [ "UUID", "filename" ],
+      "required" : [ "@context", "@type", "label", "preserve", "access", "download", "publish", "shelve" ],
       "properties" : {
-        "filename" : {
+        "@context" : {
+          "type" : "string",
+          "format": "uri",
+          "description" : "URI for the JSON-LD context definitions",
+          "example": "http://sdr.sul.stanford.edu/contexts/taco-base.jsonld"
+        },
+        "@type" : {
+          "type" : "string",
+          "format": "uri",
+          "description": "URI for the resource type of File",
+          "example" : "http://sdr.sul.stanford.edu/models/sdr3-file.jsonld"
+        },
+        "access" : {
+          "type" : "string",
+          "description": "What groups should be able to access (view) the File in Access environments"
+        },
+        "contained-by" : {
+          "type" : "string",
+          "format" : "uri",
+          "description": "The parent resource of this resource."
+        },
+        "deriv_create" : {
+          "type" : "boolean",
+          "description" : "Should derivatives be created for the File"
+        },
+        "download" : {
           "type" : "string",
-          "example" : "mybook.pdf"
+          "description": "What groups should be able to download the File in Access environments"
+        },
+        "id" : {
+          "type" : "string",
+          "description": "The TACO identifier for the resource. Parent DRUID-derived.",
+          "example" : "oo000oo0001"
```

Comment:
Should this be consistent with line 309?

Reply:
Can you elaborate? I'm not sure I understand the concern between this and 309.